tests/test_invalids.py

```diff
@@ -1168,6 +1173,11 @@ def foo(inp: num) -> bytes <= 3:
 """, TypeMismatchException)
 
 must_fail("""
+def foo() -> num:
+    return block.fail
+""", Exception)
```

Comment:
What does this have to do with exponents?

Reply:
I had an issue with coveralls so I increased test coverage to get it passing (nothing to do with exponents).

Reply:
@DavidKnott does this have to do with "decreasing" test coverage not letting you merge or something? We should disable that until `v1.0.0`